### PR TITLE
Release auto case closure

### PR DESCRIPTION
**Pull request by gcapalbo**
@dimagi/product This will remove the feature flag and make auto case closure available.  Is there anything we should wait for or can we release this?

@NoahCarnahan @snopoke code review buddies

### CODE REVIEW

**Comment by snopoke** [comment]
:boot: :construction:

**Comment by amsagoff** [comment]
@sheelio @gcapalbo would be good to have docs pre-release (and link to docs from page). Are we already doing that?

**Comment by sheelio** [comment]
Docs are done - Jeremy wrote them up: 
https://confluence.dimagi.com/display/commcarepublic/Automatic+Case+Closure

**Comment by czue** [comment]
awesome! :+1: from me but not merging due to label

File: corehq/apps/data_interfaces/tests/test_auto_case_updates.py [hunk 1]
```diff
@@ -94,3 +122,175 @@ def test_rule(self):
...
+
+        expected_rule_ids = [self.rule2.pk, self.rule3.pk]
+        actual_rule_ids = [rule.pk for rule in rules_by_case_type['test-case-type-2']]
+        self.assertEqual(set(expected_rule_ids), set(actual_rule_ids))
+
+    def test_boundary_date(self):
+        rules = AutomaticUpdateRule.by_domain(self.domain)
+        rules_by_case_type = AutomaticUpdateRule.organize_rules_by_case_type(rules)
+
+        boundary_date = AutomaticUpdateRule.get_boundary_date(rules_by_case_type['test-case-type'], datetime(2016, 1, 1))
```
Comment (dimagimon) [inline corehq/apps/data_interfaces/tests/test_auto_case_updates.py]: E501 line too long (121 > 115 characters)

File: corehq/apps/data_interfaces/tests/test_auto_case_updates.py [hunk 2]
```diff
@@ -94,3 +122,175 @@ def test_rule(self):
...
+        self.assertEqual(boundary_date, datetime(2015, 12, 2))
+
+        boundary_date = AutomaticUpdateRule.get_boundary_date(rules_by_case_type['test-case-type-2'], datetime(2016, 1, 1))
```
Comment (dimagimon) [inline corehq/apps/data_interfaces/tests/test_auto_case_updates.py]: E501 line too long (123 > 115 characters)

**Comment by gcapalbo** [comment]
@dimagi/product updated the rule list page to include a link to the docs:
auto-case-close-help-site-link

as long as you're good with that text, this should be good to merge and deploy

**Comment by amsagoff** [comment]
Proposed new text: "Create rules for closing cases from CommCareHQ. All
rules run once a week on Friday at midnight GMT. For more information, see
the Help Site"

**Comment by gcapalbo** [comment]
help text updated, this is good to go now

**Comment by czue** [comment]
killed the build

**Pull request merged**